tests/local/xds_local_test.go

```diff
@@ -190,7 +190,7 @@ func initLocalPilot(IstioSrc string) (*bootstrap.Server, error) {
 
 func startLocalPilot(s *bootstrap.Server, stop chan struct{}) {
 	// Start the server
-	_, _ = s.Start(stop)
+	s.Start(stop)
```

Comment:
Hm, I'm surprised the linter doesn't complain about this; I know the `_, _` was added to make it happy.

Reply:
Not needed